sysl2/sysl/cmd_runner.go

```diff
@@ -105,3 +121,73 @@ func EnsureFlagsNonEmpty(cmd *kingpin.CmdClause, excludes ...string) {
 
 	cmd.PreAction(fn)
 }
+
+func (r *cmdRunner) getProjectRoot(fs afero.Fs, logger *logrus.Logger) error {
```

Comment:
function naming is extremely important for code readability and maintainability. `get*()` functions are expected to return the thing it is getting. I'm struggling to figure out what this function actually does.

Reply:
changed to `setProjectRoot`